avm/utilities/pipelines/e2eValidation/resourceDeployment/New-TemplateDeployment.ps1

```diff
@@ -405,8 +405,8 @@ function New-TemplateDeployment {
         [Parameter(Mandatory = $false)]
         [string[]] $parameterFilePath,
 
-        [Parameter(Mandatory)]
-        [string] $location,
+        [Parameter(Mandatory = $false)]
+        [string] $Location = 'WestEurope',
```

Comment:
Also minor, but not sure why we made Location StartCase or if that was intended. I'd keep casing consistent with other parameters

Reply:
I did so as uppercase is the more 'correct' one. To fix this I went now the opposite, yet consistent way and updated all other occurences :)